src/main/kotlin/com/healthmetrix/labres/GlobalErrorHandler.kt

```diff
@@ -1,20 +1,30 @@
 package com.healthmetrix.labres
 
+import io.swagger.v3.oas.annotations.media.Schema
 import java.util.UUID
 import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.ControllerAdvice
 import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.ResponseStatus
 
 @ControllerAdvice
 class GlobalErrorHandler {
 
     @ExceptionHandler(Exception::class)
+    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
     fun onException(exception: Exception): ResponseEntity<InternalServerError> {
         val id = UUID.randomUUID()
         logger.error("Exception caught incidentId=$id", exception)
         return InternalServerError("$id").asEntity()
     }
 
-    data class InternalServerError(val message: String) : ApiResponse(HttpStatus.INTERNAL_SERVER_ERROR)
+    @Schema(description = "General error response body")
+    data class InternalServerError(
+        @Schema(
+            description = "A summary of the error that occurred",
```

Comment:
Right now, the content of the message is actually an incident id (UUID), should probably be mentioned here